src/main/scala/org/broadinstitute/hail/sparkextras/OrderedRDD.scala

```diff
@@ -38,72 +44,80 @@ object OrderedRDD {
   }
 
   def apply[PK, K, V](rdd: RDD[(K, V)], fastKeys: Option[RDD[K]] = None)(implicit kOk: OrderedKey[PK, K]): OrderedRDD[PK, K, V] = {
+    val (_, orderedRDD) = coerce(rdd, fastKeys)
+    orderedRDD
+  }
+
+  def coerce[PK, K, V](rdd: RDD[(K, V)], fastKeys: Option[RDD[K]] = None)
+    (implicit kOk: OrderedKey[PK, K]): (Int, OrderedRDD[PK, K, V]) = {
```

Comment:
That's what the `apply` above is for.  This lets us test freely (and maybe even helps when the query optimizer comes to town) but the previous API doesn't change.